_posts/2020-03-28-sr2020-new-provisional-date.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: news
+title: Student Robotics 2020 New Provisional Competition Date
+---
+
+## Provisional Competition Date
+
+After surveying our SR2020 teams and volunteers, we have selected the **weekend of the 4th July 2020** to be the most appropriate date for the competition.
+
+The COVID-19 situation changes often, so we will re-evaluate in mid-May.
```

Comment:
I think we need to change the wording of this slightly. As I see it, it's either this weekend, or no weekend at all (or a heavy modification of a usual competition cycle).